spec/test-data/translations/glotpress-download/filters/merged.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Translation-Revision-Date: 2021-03-31 09:54:04+0000
+Plural-Forms: nplurals=2; plural=n > 1;
+Generator: GlotPress/2.4.0-alpha
+Language: fr
+-->
+<resources>
+    <string name="gp_locale">fr</string>
+    <string name="only_in_current">This is only in "current" export</string>
+    <string name="in_multiple_exports">fuzzy</string>
+    <string name="shipping_label_woo_discount_bottomsheet_message" formatted="false">Get 5% to 40% off compared to post office rates!</string>
+    <string-array name="list">
+        <item>list_item_1_fuzzy</item>
+        <item>list_item_2_fuzzy</item>
+    </string-array>
```

Comment:
ℹ️  Ensures that (1) even when merging multiple exports for one locale, the `formatted="false"` attribute is properly reported, and that (2) merge logic also applies to `string-array` tags.